test-core.ini

```diff
@@ -31,6 +31,9 @@ ckan.datapusher.formats = csv xls xlsx tsv application/csv application/vnd.ms-ex
 ## Solr support
 solr_url = http://127.0.0.1:8983/solr/ckan
 
+# Redis URL. Use a separate Redis database for testing.
+redis_url = redis://localhost:6379/1
```

Comment:
Do we want to namespace this like newer config options?

Ex: `ckan.db.redis_uri = `?